h/views/activity.py

```diff
@@ -86,6 +80,21 @@ def group_search(request):
     if request.authenticated_user not in group.members:
         return result
 
+    def user_annotation_count(aggregation, userid):
+        for user in aggregation:
+            if user['user'] == userid:
+                return user['count']
+        return 0
+
+    q = query.extract(request)
+    users_aggregation = result.get('aggregations', {}).get('users', [])
+    members = [{'username': u.username,
+                'userid': u.userid,
+                'count': user_annotation_count(users_aggregation, u.userid),
+                'faceted_by': _faceted_by_user(request, u.username, q)}
+               for u in group.members]
+    members = sorted(members, key=lambda k: k['username'].lower())
```

Comment:
Interesting. So as the search query and matching annotations counts changed, would the order of the users change as the page reloaded? Anyway, I think alphabetical order is good, nice and consistent!